CONTRIBUTING.md

```diff
@@ -158,17 +154,26 @@ _A library project is an Angular concept for organising code that are going to b
 >
 >- When creating a release branch, follow the instructions below to the letter!
 
-1. Create a new branch from `dev`.
+1. Create a new branch from `fhi-[project]/latest`.
 2. Name it `release/fhi-[project]/x.x.x`, where `x.x.x` is the version you're releasing.
 3. Change text `# Unreleased` to `# x.x.x` in the CHANGELOG for the project: `./projects/fhi-[project]/CHANGELOG.md`
 4. (If a new line is added) Change text `Unreleased` to `x.x.x` in the dependency matrix for the project: `./projects/fhi-[project]/README.md`
 5. Change version in `./projects/fhi-[project]/package.json` to `x.x.x` manually.
     >_It's cumbersome to use `npm version` since `package.json` is in another directory than the git directory. And since there is no `package-lock.json`, and no need for a tag in the current workflow, doing it manually is faster. A better, and more automated, solution may come in the future._
-6. Create PR, and when approved, make sure commit message is the same as the branch name, except for uppercase R in Release, and then merge release branch to `dev`.
+6. Create PR, and when approved, make sure commit message is the same as the branch name, except for uppercase R in Release, and then merge release branch to `fhi-[project]/latest` (deploy).
    >_NB! Automated release job only runs if `Release/fhi-[project]/` is present in commit message since this isn't a release for everything in the repo, just a particular library._
-   >
-   >_The PR goes into `dev` because this makes the workflow much easier. The fact that `main` may be a bit behind the official history in `dev`, including a release of a library, is a small price to pay._
-7. After approved review, squash and merge to `dev` (deploy)
+
+##### Release a patch to older version in a library projects
```

Comment:
Small typo: "... older version in library projects" or "... older version in a library project"